tensorboard/components/tf_dashboard_common/dashboard-style.html

```diff
@@ -33,12 +33,26 @@
         display: flex;
         flex-direction: column;
         height: 100%;
-        margin-right: 20px;
+        margin-right: 10px;
         overflow-x: hidden;
         padding: 5px 0;
         text-overflow: ellipsis;
       }
 
+      .settings {
+        contain: layout paint;
```

Comment:
If overflow is already set to hide in x direction and scroll in y direction, is this `contain` adding anything?

Reply:
I thought it would not hurt to give browser some hint to not paint/layout out of the container in all cases but it was done without any proof that layout was causing issue. Removed.